### PR TITLE
Redesign organization card in settings

### PR DESCRIPTION
**Pull request by manuthecoder**
See #6607

### CODE REVIEW

**Comment by manu-gurudath** [comment]
Here's what it looks like so far: 

image
image

File: app/views/events/settings/_details.html.erb [hunk 1]
```diff
@@ -1,70 +1,84 @@
...
       <% if policy(@event).update? %>
-        <div class="field--fileupload">
-          <%= form.label :logo, "Choose file", class: "field--fileupload__label" %>
-          <%= form.file_field :logo, accept: "image/*,image/heic", class: "field--fileupload__field" %>
-        </div>
+       <button
+          type="button"
+          data-behavior="menu_toggle" aria-expanded="false" tabindex="0"
+          class="pop menu__toggle z1 organization_settings__banner--edit">
+          <%= inline_icon "edit", size: 24 %>
```
Comment (sampoder) [inline app/views/events/settings/_details.html.erb]: Let's use the `photo` or `photo-fill` icons here. The edit one isn't super clear for non-text inputs imo.

File: app/views/events/settings/_details.html.erb [hunk 2]
```diff
@@ -1,70 +1,84 @@
...
+          <%= form.label :description, "Mission statement" %>
+          <%= form.text_area :description, placeholder: "A short description of your organization’s goals and impacts.", disabled: !policy(@event).update? %>
+        </div>
 
-    <div class="field block">
-      <%= form.label :website, "Website URL" %>
-      <%= form.text_field :website, type: :url, placeholder: "https://hackclub.com", class: "w-100", disabled: !policy(@event).update? %>
+        <div class="field flex-1">
+          <%= form.label :website, "Website URL" %>
+          <%= form.text_field :website, type: :url, placeholder: "https://hackclub.com", disabled: !policy(@event).update? %>
```
Comment (sampoder) [inline app/views/events/settings/_details.html.erb]: Would it be possible to make these two inputs the same height?

**Review by sampoder — COMMENTED** [review]
This looks incredible; thank you so much for putting this together! Two quick comments.

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -241,6 +241,7 @@ summary {
 @import 'components/callout';
 @import 'components/reimbursements/expenses';
 @import 'components/reimbursements/actions';
+@import 'components/organization_settings';
```
Comment (malted) [inline app/assets/stylesheets/application.scss]: For consistency, what if this was named `event_settings`?

**Comment by malted** [comment]
There seems to be a layout issue on mobile - can we make the inputs wrap & stick a `ml-3` on the button?
image

**Comment by malted** [comment]
This new card looks amazing!

**Review by sampoder — APPROVED** [review]
LGTM, thank you so much for this! Once this is merged, it'd be sweet to post about this publicly

**Comment by manu-gurudath** [comment]
Cool! How would I do that?

**Comment by sampoder** [comment]
Maybe just a post in #hcb with a screenshot or a #ship post?

**Pull request merged**